updating/troubleshooting_updates/restoring-cluster-previous-state.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="restoring-cluster-previous-state"]
+= Restoring your cluster to a previous state
+include::_attributes/common-attributes.adoc[]
+:context: restoring-cluster-previous-state
+
+toc::[]
+
+If your update fails, something critical has been deleted, or if your cluster experiences quorum loss, you can restore your cluster to a previous state. 
```

Comment:
Less text to explain the redirection would be good in this case because we do not need to explain the use cases mentioned in the link again here. So my suggestion would be just delete this text in the next paragraph `For information on how to restore your cluster to a previous state, see` is good enough for this.